<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>handGesture</class>
 <widget class="QWidget" name="handGesture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>handGesture</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>650</width>
     <height>260</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera</string>
   </property>
   <widget class="QLabel" name="label_srcScreen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;原始图像&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cvtScreen">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;识别图像&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>171</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Recongnize Result</string>
   </property>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;识别结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Operation Area</string>
   </property>
   <widget class="QPushButton" name="pushButton_openCamera">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startRecognize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Digit Recognize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_captureImage">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DirectionRecognize">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dir Recognize</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
